include/exadg/acoustic_conservation_equations/spatial_discretization/interface.h

```diff
@@ -31,10 +31,11 @@ namespace Acoustics
 {
 namespace Interface
 {
-template<typename Number>
+template<int dim, typename Number>
```

Comment:
Why `dim`?

A main reason for the existence of these interface classes is the goal to write time integration classes without template parameter `dim`.